.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,29 @@
+name: Spellcheck
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  spellcheck:
+    name: Spelling
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+        show-progress: false
+    - uses: codespell-project/actions-codespell@master
+      with:
+#        builtin: clear,en-GB_to_en-OED,en-US_to_en-OED
+        path: dat/missions/
```

Comment:
Would probably make sense to do events too.
Doing something like outfits / ships would be more complex as it is XML and not all fields are actually used as display strings. We do have a list of which are useful in the translation files, but not sure if it's easy to use directly in this case.